src/js/views/maps/viewfinder/ViewfinderView.js

```diff
@@ -216,6 +184,34 @@ define(
         this.getList().hide();
       },
 
+      /**
+       * Render the SearchInputView. 
+       */
+      renderSearchInput() {
+        this.searchInput = new SearchInputView({
+          placeholder: "Enter coordinates or areas of interest",
+          search: text => {
+            this.viewfinderModel.search(text);
+            // TODO(ianguerin): set error if results are empty.
```

Comment:
Is this for future or a reminder for this PR?

Reply:
No, just an out of date Todo, removing